### DISCUSSION TITLE
FB Noscript code correction?

### DISCUSSION THREAD

**Discussion by ellismp** [manucaralmo/GlowCookies#40] [General]
Hi unless I'm missing something I think lines 151++ should be something like this otherwise the IMG tag is missing

      let FacebookPixelNS = document.createElement('noscript');
      let FacebookPixel = document.createElement('img');
      FacebookPixel.setAttribute('height', `1`);
      FacebookPixel.setAttribute('width', `1`);
      FacebookPixel.setAttribute('style', `display:none`);
      FacebookPixel.setAttribute('src', `https://www.facebook.com/tr?id=${this.tracking.FacebookPixelCode}&ev=PageView&noscript=1`);
      FacebookPixelNS.appendChild(FacebookPixel);
      document.head.appendChild(FacebookPixelNS);

Best
Martin

**Comment by manucaralmo** [comment]
Yes, I think you are right @ellismp. I will check it and we will fix it
